src/hyp3_isce2/insar_stripmap.py

```diff
@@ -24,31 +33,89 @@
     os.environ['PATH'] = str(ISCE_APPLICATIONS) + os.pathsep + os.environ['PATH']
 
 
-def insar_stripmap(reference_scene: str, secondary_scene: str) -> Path:
-    """Create an interferogram
-
-    This is a placeholder function. It will be replaced with your actual scientific workflow.
+def insar_stripmap(user: str, password: str, reference_scene: str, secondary_scene: str) -> Path:
+    """Create a Stripmap interferogram
```

Comment:
Various parts of this workflow assume that you are working with ALOS-1 data, even though `stripmapApp` can work with other datasets. I don't think this is a problem, but we should put something in the docstring here telling users that this is the case.

Reply:
At the moment, the only products from asf that I've been able to process with stripmap are the ALOS images. The products from ERS, Radarsat from asf are not compatible with ISCE for now.